cardano-explorer-db/src/Explorer/DB/Query.hs

```diff
@@ -140,7 +141,7 @@ queryLatestSlotNo = do
   pure $ fromMaybe 0 (listToMaybe $ mapMaybe unValue res)
 
 -- | Get the current total supply of Lovelace.
-queryTotalSupply :: MonadIO m => ReaderT SqlBackend m Word64
+queryTotalSupply :: MonadIO m => ReaderT SqlBackend m Micro
```

Comment:
Why do we now have two versions of this function, with the same name, one in my code and one on your code?

DRY (don't repeat yourself) is an important concept in software engineering.

I suggest that you rename yours to `queryTotalSupplyAda` and have it call mine, doing a conversion from lovelace to Ada.

Reply:
These is even a compelling argument to be made that this version should be the only version and replace the one in my code.